src/sharedTypes.ts

```diff
@@ -42,3 +42,9 @@ export interface Clip {
 }
 
 export type MediaFileExtension = AudioFileExtension | VideoFileExtension;
+
+export enum OperatingSystems {
```

Comment:
This will store the values as numbers, can we instead do like Linux = 'LINUX' etc so the runtime values are human readable

Also, maybe say MacOS instead of Darwin, Darwin is just the string identifier and not many people are familiar with it

Reply:
human readable refactor done here: https://github.com/patrickbrett/mlvet/pull/38/commits/d0e83b46c50d1224dbed8e815b067fd19d3ae308